File: java/src/main/java/com/experian/dvs/client/server/RestApiResponseError.java
package com.experian.dvs.client.server;

import com.fasterxml.jackson.annotation.JsonProperty;

public class RestApiResponseError {

    @JsonProperty("type")
    private String type;

    @JsonProperty("title")
    private String title;

    @JsonProperty("detail")
    private String detail;

    @JsonProperty("instance")
    private String instance;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDetail() {
        return detail;
    }

    public void setDetail(String detail) {
        this.detail = detail;
    }

    public String getInstance() {
        return instance;
    }

    public void setInstance(String instance) {
        this.instance = instance;
    }
}

File: csharp/DVSClient/DVSClient/server/RestApiResponseError.cs
using Newtonsoft.Json;

namespace DVSClient.Server
{
    public class RestApiResponseError
    {
        [JsonProperty("type")]
        public string? Type { get; set; }

        [JsonProperty("title")]
        public string? Title { get; set; }

        [JsonProperty("detail")]
        public string? Detail { get; set; }

        [JsonProperty("instance")]
        public string? Instance { get; set; }
    }
}
